### PR TITLE
tests: Add unit tests

### PR DESCRIPTION
**Pull request by animeshk08**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation

> /kind failing-test
> /kind feature
> /kind flake

 /kind test
**What this PR does / why we need it**:
Adds unit tests to improve test coverage
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Release note**:
```
Add unit tests for: azurefile, csi-common/utils, controllerserver-default, identityserver 
```

### CODE REVIEW

**Comment by animeshk08** [comment]
Partially fixes the issue: #258

**Comment by animeshk08** [comment]
Hi @andyzhangx. I have checked the log but I am unsure why the test has failed. Can you please help me?

File: pkg/azurefile/azurefile_test.go [hunk 1]
```diff
@@ -27,6 +29,28 @@ import (
...
+
+func NewFakeDriver() *Driver {
+	driver := NewDriver(fakeNodeID)
+	driver.Name = fakeDriverName
+	driver.Version = vendorVersion
+	return driver
+}
+
+func TestNewFakeDriver(t *testing.T) {
+	// Test New fake driver.
```
Comment (andyzhangx) [inline pkg/azurefile/azurefile_test.go:49]: remove this comment since it's not necessary

**Comment by animeshk08** [comment]
Hi, @andyzhangx Thank you for the reviews. I have made the changes but the tests are still failing.

**Comment by andyzhangx** [comment]
> Hi, @andyzhangx Thank you for the reviews. I have made the changes but the tests are still failing.

looks like it's a transient issue. thanks. test coverage is now:
coverage/coveralls — Coverage increased (+6.6%) to 32.014%

**Comment by animeshk08** [comment]
Thank you for the help. I will try and add more tests :)

**Pull request merged**